csharp/src/Ice/Reference.cs

```diff
@@ -1038,13 +1052,18 @@ void TraceRetry(string message, int attempt, RetryPolicy policy, Exception? exce
         }
 
         // Helper constructor for fixed references. Uses the communicator's defaults.
-        internal Reference(Communicator communicator, Connection fixedConnection, Identity identity)
+        internal Reference(
+            Communicator communicator,
+            Connection fixedConnection,
+            Identity identity,
+            IReadOnlyList<InvocationInterceptor>? invocationInterceptors) // already a copy provided by Ice
```

Comment:
I don't think you need to add it here. The defaults used by this convenience ctor is an empty invocation interceptor list.

Reply:
Fixed